src/sparseml/pytorch/torchvision/train.py

```diff
@@ -81,13 +83,24 @@ def train_one_epoch(
         start_time = time.time()
         image, target = image.to(device), target.to(device)
         with torch.cuda.amp.autocast(enabled=scaler is not None):
-            output = model(image)
+            outputs = output = model(image)
```

Comment:
Is this a typo or is the output getting assigned to two vars on purpose?

Reply:
Damn that is clever, nice! 🚀